### PR TITLE
Document lack of multiprocess / async functionality

### PR DESCRIPTION
**Pull request by Zen-CODE**
For further reference:

https://github.com/kivy/kivy-ios/issues/372

https://github.com/kivy/kivy-ios/issues/361

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,16 @@ It is due to invalid archs, search for them and check it. Maybe you
 targetted a simulator but have only armv7/arm64. Maybe you want to target
 your iPad but it as only x86_64.
 
+### Why does the python multiprocess module/async call not work?
```
Comment (misl6) [inline README.md]: Why also `async` ? 
Seems that asyncio own subprocess module only imports `PIPE`, `STDOUT`, `DEVNULL` from subprocess (https://github.com/kivy/kivy-ios/issues/372#issuecomment-534253980)
I would change this one to **"Why does the python multiprocess/subprocess module not work?"**

File: README.md [hunk 2]
```diff
@@ -249,6 +249,16 @@ It is due to invalid archs, search for them and check it. Maybe you
...
+
+The iOS application model does not currently support multi-processing in a
+cross-platform compatible way. The application design focuses on minizing
+processor usage (to minimize power consumption) and promtoes an
+[alternative conurrency model](https://developer.apple.com/library/archive/documentation/General/Conceptual/ConcurrencyProgrammingGuide/Introduction/Introduction.html).
+
+If you need to make use of threads or processes, you should consider using
```
Comment (misl6) [inline README.md]: "**python** threads" are fine in `kivy-ios` packaged apps

File: README.md (hunk 1 above)
Comment (Zen-CODE) [reply README.md]: Agreed. Done.

File: README.md [hunk 3]
```diff
@@ -249,6 +249,16 @@ It is due to invalid archs, search for them and check it. Maybe you
 targetted a simulator but have only armv7/arm64. Maybe you want to target
 your iPad but it as only x86_64.
 
+### Why does the python multiprocess/subprocess module not work?
+
+The iOS application model does not currently support multi-processing in a
+cross-platform compatible way. The application design focuses on minizing
```
Comment (misl6) [inline README.md]: Just a minor typo: "minizing", then LGTM. (Maybe a `squash` before merging it)

Comment (Zen-CODE) [reply README.md]: Done. Easiest to squash when you merge. There is that option is the github merge options? Thanks. Great to be thorough :+1:

**Pull request merged**